appinfo/info.xml

```diff
@@ -62,6 +62,7 @@ And in the works for the [coming versions](https://github.com/nextcloud/spreed/m
 		<job>OCA\Talk\BackgroundJob\CheckReferenceIdColumn</job>
 		<job>OCA\Talk\BackgroundJob\CheckHostedSignalingServer</job>
 		<job>OCA\Talk\BackgroundJob\CheckMatterbridges</job>
+		<job>OCA\Talk\BackgroundJob\ApplyTtl</job>
```

Comment:
Remove this, it's not needed as each room will get their own job added on the fly

Reply:
Good to known. I will remove.